x-pack/plugins/cloud_integrations/cloud_experiments/README.mdx

```diff
@@ -0,0 +1,177 @@
+---
+id: kibCloudExperimentsPlugin
+slug: /kibana-dev-docs/key-concepts/cloud-experiments-plugin
+title: Cloud Experiments service
+description: The Cloud Experiments Service provides the necessary APIs to implement A/B testing scenarios, fetching the variations in configuration and reporting back metrics to track conversion rates of the experiments.
+date: 2022-09-07
+tags: ['kibana', 'dev', 'contributor', 'api docs', 'cloud', 'a/b testing', 'experiments']
+---
+
+# Kibana Cloud Experiments Service
+
+The Cloud Experiments Service provides the necessary APIs to implement A/B testing scenarios, fetching the variations in configuration and reporting back metrics to track conversion rates of the experiments.
```

Comment:
nit: Maybe worth being super clear this is only active on cloud? (in case it isn't obvious from the name)

```suggestion
The Cloud Experiments Service provides the necessary APIs to implement A/B testing scenarios, fetching the variations in configuration and reporting back metrics to track conversion rates of the experiments.

The `cloudExperiments` plugin is disabled by default and only enabled on Elastic Cloud deployments.
```